static_frame/core/store_filter.py

```diff
@@ -105,11 +103,16 @@ def __init__(self, *,
         self.from_posinf = from_posinf
         self.from_neginf = from_neginf
 
-        self.to_nan = to_nan
-        self.to_nat = to_nat
-        self.to_none = to_none
-        self.to_posinf = to_posinf
-        self.to_neginf = to_neginf
+        def sanitize(c: tp.Collection[str]) -> tp.FrozenSet[str]:
+            if not isinstance(c, frozenset):
+                return frozenset(c)
+            return c
```

Comment:
Maybe lets just put this function in `util.py`, calling it `frozenset_filter()` or something similar.